### PR TITLE
DENG-4047 Don't wrap metrics in JSON in accounts events stream tables

### PR DESCRIPTION
**Pull request by akkomar**
Accounts telemetry makes heavy use of ping-level metrics with events. We would like to have an explore in Looker that would allow to use these metrics.
Events stream is a nice model to explore events because of clustering per event category, but it would be easier to work with in both Looker and BQ console if metrics weren't wrapped as JSON.
This PR makes metrics a nested record in `events_stream` for `accounts_frontend` and `accounts_backend`. This will require manual deletion of these two tables before next or of these queries.

As next steps I propose to:
1. After I confirm that Looker explores affected by this change work correctly, switch `events_stream` to store metrics as struct by default for newly created tables.
2. Switch existing tables to the new approach, taking into account backfills and cross-channel views.

I'd like to do this in stages since step 2. will require some caution and a) I am open to comments if this should be applied by default to all tables (IMO it makes sense), b) we're under a little time pressure with some Accounts dashboarding work that this PR unblocks.

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-4222)

### CODE REVIEW

File: sql_generators/glean_usage/templates/events_stream_v1.query.sql [hunk 1]
```diff
@@ -91,8 +91,11 @@ WITH base AS (
         ping_info.end_time,
         ping_info.parsed_end_time,
         ping_info.ping_type
-      ) AS ping_info,
+      ) AS ping_info
+      {% if not metrics_as_struct %}
+      ,
```
Comment (quiiver) [inline sql_generators/glean_usage/templates/events_stream_v1.query.sql:96]: bigquery sql is fine with trailing commas https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#trailing_commas

Comment (akkomar) [reply sql_generators/glean_usage/templates/events_stream_v1.query.sql:96]: Yeah, but in the SELECT statement. This is inside of REPLACE (...) and is not allowed (and painful to read).

**Review by quiiver — APPROVED** [review]
i don't love special casing specific events stream tables. if this works well for your explores lets consider moving all events stream tables to structured metrics

**Comment by quiiver** [comment]
oh i just re-read the pr description and saw this
>    Switch existing tables to the new approach, taking into account backfills and cross-channel views.

**Pull request merged**